build-all.sh

```diff
@@ -9,8 +9,8 @@ ROOT_DIR=${PWD}
 
 if [ -z "$*" ]
 then
-  MVN_ARGS="-Pscala-2.11.x -Peclipse-luna clean install"
-  MVN_P2_ARGS="-Dtycho.localArtifacts=ignore -Pscala-2.11.x -Peclipse-luna clean verify"
+  MVN_ARGS="-Pscala-2.11.x -Peclipse-mars $ADDITIONAL_MVN_OPTS clean install"
+  MVN_P2_ARGS="-Dtycho.localArtifacts=ignore -Pscala-2.11.x -Peclipse-mars $ADDITIONAL_MVN_OPTS clean verify"
```

Comment:
Instead of the env varible you could just pass `"$@"` at the end, that would forward all parameters from `build-all.sh` to maven.